app/models/concerns/hostext/search.rb

```diff
@@ -11,27 +11,27 @@ module Search
 
       scoped_search :on => :name,          :complete_value => true, :default_order => true
       scoped_search :on => :last_report,   :complete_value => true, :only_explicit => true
-      scoped_search :on => :comment,       :complete_value => true
+      scoped_search :on => :comment,       :complete_value => true, :only_explicit => true
```

Comment:
when using free text i would expect comment to be searched?

Reply:
this doesn't seem to have a significant effect in tests, reverted.